sherpa/astro/io/xstable.py

```diff
@@ -115,6 +115,11 @@
    mygauss.pos  thawed            1            0          2.4
    mygauss.norm thawed            1            0        1e+24
 
+.. testcleanup::
+
+   >>> import os
+   >>> os.remove("example.mod")
```

Comment:
I have wondered where/why some of these files keep popping up in my tree during development! (there's another recent PR with a similar change).

So thanks for the fixes.